doc/contributing/advocacy-ambasador-program.md

```diff
@@ -0,0 +1,92 @@
+# Advocacy Ambassador program
+
+Node.js aims to support and encourage developers and developer
+advocates to promote Node.js as a project and to help communicate
+key messages to their constituencies.
+
+The ambassador program does that by:
+
+* Maintaining an up-to-date set of messages that it would like ambassadors
```

Comment:
What does "it" refer to here? Presumably "the ambassador program" but if so, this is a really weird thing because "the ambassador program" would seem to include ambassadors, but they're not the ones maintaining that list, so.... ???? Should "it" say "the TSC" or "Node.js maintainers" or something like that?

Reply:
it refers to the project. Happy to make it Node.js maintainers.

Reply:
updated